tests/BenchmarkDotNet.Tests/Portability/ProcCpuInfoParserTests.cs

```diff
@@ -0,0 +1,436 @@
+﻿using BenchmarkDotNet.Helpers;
+using Xunit;
+
+namespace BenchmarkDotNet.Tests.Portability
+{
+    public class ProcCpuInfoParserTests
+    {
+        [Fact]
+        public void TestEmpty()
```

Comment:
In other test classes, we *usually* put `Test` at the end of a test method (e.g. `EmptyTest` instead of `TestEmpty`). Please, follow the convention.